addons/hr_contract/models/__init__.py

```diff
@@ -3,3 +3,4 @@
 
 from . import hr_contract
 from . import res_users
+from . import resource
```

Comment:
Careful hr_contract doesn't depends on resource

Reply:
hr does depend on resource :wink:

Reply:
Ok then it is not a problem, my bad